Traceback (most recent call last):
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 146, in <module>
    main()
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 134, in main
    process_json_and_shapes(
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 119, in process_json_and_shapes
    save_response(output_dir, idx, response, entity_ids)
TypeError: save_response() takes 3 positional arguments but 4 were given
